app/views/events/index.slim

```diff
@@ -1,11 +1,13 @@
-h2 Events
 -@events.each do |event|
-  = link_to event.name, event_path(event.id)
-  ul
-    li 
-      b = "On #{Date::DAYNAMES[event.start_date.wday]}"
-    li = event.starting_date
-    li = event.ending_date
-      li.fa.fa-user = "#{event.attendees.size}"
+  .Event
+    = link_to event.name, event_path(event.id), class: 'Event-title'
+    ul
+      li.Event-info
+        b = "On #{Date::DAYNAMES[event.start_date.wday]}"
```

Comment:
What's the `b`. Currently it's only used for styling. So if that's the case here, you should do it with CSS. If it's for emphasis you have `em`